src/multiharp_toolkit/ptu_parser.py

```diff
@@ -98,12 +99,17 @@ def parse_record(self, data: int):
             truetime = self.oflcorrection + timetag
             self.append_events(channel + 1, truetime)
 
-    def append_events(self, channel: int, timestamp: float):
+    def timetag2time(self, timetag: int) -> int:
+        """convert time tag to time(unit: psec)
```

Comment:
Avoid abbreviations.

Avoid vagueness.

```suggestion
        """Convert a time tag to a relative timestamp with picosecond resolution.
```

Reply:
I think it is probably good.
However, I want to tell the unit of the number returned from this method has picosecond.
I suppose "timestamp with picosecond resolution" means "this timestamp has resolution of 1 picosecond", however, I do not expect this mean.

What I want to inform in this sentense is , for example, simply 10 means 10 picosecond, and 1000 means 1000 picosecond or 1 nano second.
How should I represent in this case?

Reply:
> What I want to inform in this sentense is , for example, simply 10 means 10 picosecond, and 1000 means 1000 picosecond or 1 nano second.
How should I represent in this case?

The word "resolution" usually means "10 means 10 picoseconds". For example, [it's used that way in the Postgres documentation](https://www.postgresql.org/docs/current/datatype-datetime.html). "With picosecond resolution" also has this meaning.

However, we can make this even simpler:

```suggestion
        """Convert a time tag to a relative timestamp in picoseconds.
```

Reply:
>The word "resolution" usually means "10 means 10 picoseconds".

I see.

>I suppose "timestamp with picosecond resolution" means "this timestamp has resolution of 1 picosecond", however, I do not expect this mean.
This is my mis-understanding.

So I think first one is good.
Thank you.